tre_schemas/avro/request-courtDocument-parse.avsc

```diff
@@ -0,0 +1,60 @@
+{
+  "type": "record",
+  "name": "RequestCourtDocumentParse",
+  "namespace": "uk.gov.nationalarchives.tre.messages.request.courtdocument.parse",
+  "doc": "properties required for all messages routed through TRE",
+  "fields": [
+    {
+      "name": "properties",
+      "type": "uk.gov.nationalarchives.tre.messages.event.Properties"
+    },
+    {
+      "name": "parameters",
+      "doc": "message specific parameters required by the consumer to be be able to process event",
+      "type": {
+        "type": "record",
+        "name": "Parameters",
+        "fields": [
+          {
+            "name": "s3Bucket",
+            "type": "string",
+            "doc": "s3 bucket containing the document to be parsed"
+          },
+          {
+            "name": "s3Key",
+            "type": "string",
+            "doc": "s3 key of the document to be parsed"
+          },
+          {
+            "name": "reference",
+            "type": "string",
```

Comment:
This refers specifically to FCL, as does the PR name, whereas originator refers to FCL/TDR.

Just wanted to clarify whether this request schema is going to be the same for all parser requests, be it from TDR or FCL etc.

Reply:
currently this is only going to be used by you. In future anybody may use it, possibly the preservation system will want to preserve updated parsed versions

Reply:
Right, so for now, the specific reference to FCL is fine, as long as other systems can't send messages to TRE v2 without you allowing that in the policies.